### ISSUE TITLE
URGENT: Bot not working under linux

### ISSUE THREAD

**Issue by schuesslerf** [freebitcobot/freebitcobot#9]
Did anybody got the bot working under linux? I tried it with Ubuntu 18.10, Ubuntu 14.04, CentOS 7 and Debian 8 now, and none of these worked (refering to the Appimage thing), also I tried to build it myself, but it always get back to the same problem, which seems to be something called "robot js"... When trying to execute the AppImage I can a Stacktrace which leads back to robot.js and when I try to build it myself respectively when I try to run "yarn", theres also an error regarding robot.js...

Can anybody help me or can the the developer fix this?

**Comment by maxcado** [comment]
Yeah, not working in linux. I get similar error regarding robot.js on Ubuntu plus other module errors. Also, there seems to be an issue in Windows with captcha. I keep getting a message that captcha has expired. Also, which file do I make changes regarding a captcha api?

**Comment by kwuite** [comment]
Got it working on Ubuntu 18.04. I successfully update the packages and build the app image. Anyone still needs this?

**Comment by maxcado** [comment]
@kwuite Sure dude. Is it available now for download?
